src/components/Plan.js

```diff
@@ -33,30 +34,44 @@ class Plan extends Component {
 
         let phasedIssues = {};
         phases.forEach(phase => {
-            phasedIssues[phase] = this.props.issues.filter(issue => {
-                return issue.labels.map(label => label.name).includes(phase);
+            phasedIssues[phase] = {};
+            repos.forEach(repo => {
+                phasedIssues[phase][repo] = this.props.issues.filter(issue => {
+                    // FIXME: repo should either mean "vector-im/riot-web" or "riot-web"
+                    // consistently
+                    return (issue.labels.map(label => label.name).includes(phase)
+                        && (issue.owner + '/' + issue.repo) === repo );
+                });
             });
         });
 
         // Add unphased issues as an extra section
-        phasedIssues["unphased"] = this.props.issues.filter(issue => {
-            const labels = issue.labels.map(label => label.name);
-            return !labels.some(label => label.startsWith("phase:"));
+        phasedIssues["unphased"] = {};
+        repos.forEach(repo => {
+            phasedIssues["unphased"][repo] = this.props.issues.filter(issue => {
+                const labels = issue.labels.map(label => label.name);
+                // FIXME: repo should either mean "vector-im/riot-web" or "riot-web"
+                // consistently
+                return (!labels.some(label => label.startsWith("phase:"))
+                    && (issue.owner + '/' + issue.repo) === repo);
+            });
+            if (phasedIssues["unphased"][repo].length > 0) {
+                phases.push("unphased");
```

Comment:
If multiple repos have unphased issues, I believe this would add `unphased` to the `phases` array multiple times (since it runs for each repo).

One possible solution would be to use a `let hasUnphased = false` before the repo loop, set that to `true` for any repo that has them, and then after the repo loop add `unphased` to the `phases` if needed.

Reply:
I'm not sure if this is still a problem; probably not.

Reply:
It seems like this is fixed now. 🎉